### PR TITLE
HAWKULAR-758 - Added cassandra driver as WF module

### PR DESCRIPTION
**Pull request by jpkrohling**

### CODE REVIEW

**Comment by jpkrohling** [comment]
For reference, see hawkular/hawkular#617

File: cassandra-driver-wf-module/src/main/module-resources/main/module.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<module xmlns="urn:jboss:module:1.1" name="com.datastax.cassandra">
+  <dependencies>
+    <module name="com.google.guava"/>
+    <module name="org.slf4j"/>
+    <module name="io.netty"/>
+    <module name="sun.jdk"/>
```
Comment (tsegismont) [inline cassandra-driver-wf-module/src/main/module-resources/main/module.xml:25]: What does this module provide?

File: cassandra-driver-wf-module/src/main/module-resources/main/module.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<module xmlns="urn:jboss:module:1.1" name="com.datastax.cassandra">
+  <dependencies>
```
Comment (tsegismont) [inline cassandra-driver-wf-module/src/main/module-resources/main/module.xml:21]: I guess you checked the version of the libraries exposed match the version needed by the C\* driver?

**Comment by tsegismont** [comment]
LGTM, waiting for answers. I can't merge myself though, I don't have proper permissions.

File: cassandra-driver-wf-module/src/main/module-resources/main/module.xml (hunk 1 above)
Comment (jpkrohling) [inline cassandra-driver-wf-module/src/main/module-resources/main/module.xml:25]: The driver needs sun.misc.Unsafe, which is provided on this module.

File: cassandra-driver-wf-module/src/main/module-resources/main/module.xml (hunk 2 above)
Comment (jpkrohling) [inline cassandra-driver-wf-module/src/main/module-resources/main/module.xml:21]: Yes. Basically, I started with all transitive dependencies from the driver (which in this case was only io.netty, slf4j and guava). We had the same versions on Wildfly already, so, I just replaced by the module dependencies.

**Pull request merged**